react/features/base/styles/functions.web.ts

```diff
@@ -32,7 +32,7 @@ export function getFixedPlatformStyle(style: StyleType): StyleType {
  * @param {Object} base - The base object containing the `lineHeight` property.
  * @returns {Object}
  */
-export function withPixelLineHeight(base: any): Object {
+export function withPixelLineHeight(base: any) {
```

Comment:
Because it was useless. In TypeScript using the Object type is not recommended because it does not mean much. Instead of having a type that does pretty much nothing it's better to leave it empty and have TS figure out the return type